src/confcom/azext_confcom/custom.py

```diff
@@ -258,14 +278,30 @@ def acifragmentgen_confcom(
         policy = security_policy.load_policy_from_config_file(
             input_path, debug_mode=debug_mode, disable_stdio=disable_stdio
         )
+    # get all of the fragments that are being used in the policy
+    # and associate them with each container group
+    fragment_policy_list = []
+    container_names = []
+    fragment_imports = policy.get_fragments()
```

Comment:
should we log if `fragment_imports` is empty?

Reply:
I don't think so, it doesn't change functionality and should be a relatively common scenario